src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -830,7 +830,7 @@ RCLConsensus::timerEntry(NetClock::time_point const& now)
     catch (SHAMapMissingNode const& mn)
     {
         // This should never happen
-        JLOG(j_.error()) << "Missing node during consensus process " << mn;
+        JLOG(j_.error()) << "During consensus process: " << mn.what();
```

Comment:
To determine which function caught, consider adding the function name to the log.
eg.
```JLOG(j_.error()) << "During consensus process: " << mn.what() << ". Caught in " << __func__;```

Reply:
I can't wait for C++20 and [`std::source_location`](https://en.cppreference.com/w/cpp/utility/source_location). :)

Reply:
Personally I wouldn't dare to write such a proposal until I had some field experience with it.

Reply:
I can hear the cries of ABI compatibility already.  A change like that would need to be more carefully crafted than I know how to do.

Reply:
Appreciate both of your feedback. :vulcan_salute: